index.html

```diff
@@ -25,6 +25,20 @@ <h1 id="logo">Aziz Mejri</h1>
           </li>
         </ul>
       </nav>
+
+      <nav id="mobile-menu-container">
+        <div class="menu-shape">
+          <img class="img-square" src="./media/svg/square.svg" alt="square" />
+          <i id="mobile-close-menu" class="fa fa-times"></i>
+        </div>
+        <ul>
+          <li><a class="nav-link" href="#">Hello</a></li>
+          <li><a class="nav-link" href="#projects">Portoflio</a></li>
+          <li><a class="nav-link" href="#about-me">About</a></li>
+          <li><a class="nav-link" href="#contact">Contact</a></li>
```

Comment:
As you can see in the original template:
![image](https://user-images.githubusercontent.com/28903404/139599572-0e882063-448e-446a-a0d2-0fdf86e7884d.png)
The links have an `underline`! you can achieve this effect in multiple ways! let me suggest you some:
- [Text underline](https://developer.mozilla.org/en-US/docs/Web/CSS/text-decoration)
- [Border bottom](https://www.w3schools.com/CSSref/pr_border-bottom.asp)
- Or if you are looking for something different you can try to dray the underline with: [pseudo elements](https://www.w3schools.com/CSS/css_pseudo_elements.asp)

Remember that it is important to follow the given template and that is the reason I am asking for this change :rocket:

Reply:
FIxed